RaftMerge/RaftMerge.tla

```diff
@@ -40,22 +57,38 @@ VARIABLES messages
 
 \* The data structures in C. MAXS = |Store|.
 \*
+\* enum Log { LogNormal, LogPreMerge, LogMerge };
+\*
+\* enum RegionState { RegionNormal, RegionTombStone, RegionMerging };
+\*
 \* struct Raft {
 \*   bool is_leader;
 \*   vector<Log> logs;
 \*   int commit_index;
 \*   int apply_index;
+\*   int num_applied;        // number of applied normal logs
```

Comment:
can we use apply_index instead of it?

Reply:
No. We can skip some committed logs when applying them. It is necessary to distinguish these skipped logs and do-applied logs.